### ISSUE TITLE
TV3 - ERROR: Something wrong with that url

### ISSUE THREAD

**Issue by Sopor-** [spaam/svtplay-dl#138]
When i was trying to download Efterlyst i got this error message.

```
svtplay-dl -S -A http://www.tv3play.se/program/efterlyst
INFO: Outfile: efterlyst-avsnitt-1-tv3-play.ts
[031/360][=========..........................................................................................................] ETA: 0:01:30ERROR: Something wrong with that url
ERROR: Error code: 404
```

Btw, can you move the error message one row down so it's easier to read the error message?

**Comment by selepo** [comment]
Should this be closed then ? (seems to be fixed in #146 and I can't reproduce the error)

**Comment by dentarg** [comment]
I think the comment from above is enough to close this
